spanner/ddl/ast.go

```diff
@@ -155,7 +181,11 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 		if i < len(ct.ColNames)-1 {
 			s += ","
 		} else {
-			s += " "
+			if len(ct.Fks) > 0 && config.ForeignKeys {
```

Comment:
These change seems leftover from when we used to print foreign keys as part of create table statements?
Delete this change (lines 184-188) and revert to previous code.

Reply:
yes, removed this changes